src/styles/header.scss

```diff
@@ -0,0 +1,84 @@
+@import './variables';
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  width: 100%;
+  max-width: 100%;
+  min-width: 1024px;
+  display: flex;
+  justify-content: space-between;
+  position: relative;
+  align-items: center;
+  background-color: rgb(255, 255, 255);
+  padding: 0 50px;
+  box-sizing: border-box;
+
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
The `font-family` property is better to be set in a more global scope, for example, in `body` or `html` tag. This way, you don't have to specify it in each component or element. This helps maintain consistency and reduces the amount of code.